docs/en/observability/ingest-uptime.asciidoc

```diff
@@ -0,0 +1,165 @@
+[chapter, role="xpack"]
+[[ingest-uptime]]
+
+:beatname_uc: Heartbeat
+:beatname_lc: heartbeat
+:beatname_pkg: heartbeat-elastic
+:beatname_url: {beats-ref-root}/{beatname_lc}/{branch}
+
+[[ingest-uptime]]
+= Ingest uptime data
+
+[NOTE]
+=====
+If you haven't already, you need to install {es} for storing and searching your data, and {kib} for visualizing and
+managing it. For more information, see <<install-observability,Get started>>.
+=====
+
+[[install-heartbeat]]
+== Step 1: Install {beatname_uc}
+
+To download and install {beatname_uc}, use the commands that work with your system:
```

Comment:
Should we mention again that hearthbeat should be installed on a different host than the one it's monitoring?

Reply:
Yes, good point.